x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page_overview/status_details.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent, ReactNode } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiBadge, EuiFlexGroup, EuiFlexItem, EuiIcon, EuiText, EuiTextColor } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { euiThemeVars } from '@kbn/ui-theme';
+
+import { useAppContext } from '../../../../../app_context';
+import { Index } from '../../../../../../../common';
+import { OverviewCard } from './overview_card';
+
+export const StatusDetails: FunctionComponent<{
+  documents: Index['documents'];
+  documentsDeleted: Index['documents_deleted'];
+  status: Index['status'];
+  health: Index['health'];
+}> = ({ documents, documentsDeleted, status, health }) => {
+  const { config } = useAppContext();
+  if (!config.enableIndexStats) {
+    return null;
+  }
+  let healthBadge: ReactNode;
```

Comment:
I know its a bit out of scope for this PR so feel free to ignore, but dont we have more information on why the health status might be critical or warning? Feels like we are telling things to the user but they might not know what caused it or how to fix it

Reply:
Good point, opened https://github.com/elastic/kibana/issues/170040 to track this